<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian trueg</author>
 <class>base_K3bMovixOptionsWidget</class>
 <widget class="QWidget" name="base_K3bMovixOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Playback Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>MPlayer subtitle fontset:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="m_comboSubtitleFontset">
        <property name="toolTip">
         <string>Select the font to be used to render subtitles</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="textLabel7">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Unwanted MPlayer options:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="m_checkRandomPlay">
          <property name="toolTip">
           <string>The files are played in random order</string>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;If this option is checked the order in which the files are played is determined randomly every time it is played.</string>
          </property>
          <property name="text">
           <string>Pla&amp;y files randomly</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkNoDma">
          <property name="toolTip">
           <string>Do not use DMA for media access</string>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;If this option is checked the resulting eMovix CD/DVD will not use DMA for accessing the drive. This will slow down reading from the CD/DVD but may be necessary on some systems that do not support DMA.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Do not use DMA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="m_editUnwantedMplayerOptions">
        <property name="toolTip">
         <string>MPlayer options you want to be sure MPlayer will not use</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;Here one can specify MPlayer options that should never be used.
&lt;p&gt;They have to be separated by spaces:
&lt;pre&gt;opt1 opt2 opt3&lt;/pre&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="m_editAdditionalMplayerOptions">
        <property name="toolTip">
         <string>Additional MPlayer options</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;MPlayer options that should be used in any case.
&lt;p&gt;They have to be separated by spaces:
&lt;pre&gt;opt1 opt2 opt3&lt;/pre&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel6">
        <property name="text">
         <string>Additional MPlayer options:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Loop playlist:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_spinLoop">
        <property name="toolTip">
         <string>How many times should the playlist be looped</string>
        </property>
        <property name="specialValueText">
         <string>infinity</string>
        </property>
        <property name="suffix">
         <string> time(s)</string>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_labelAudioBackground">
        <property name="text">
         <string>Audio Player Background:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KComboBox" name="m_comboAudioBackground">
        <property name="toolTip">
         <string>Background video to show during audio playback</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;&lt;b&gt;Audio Player Background&lt;/b&gt;
&lt;p&gt;During audio playback normally the screen would be black. However, if a background movie has been selected, eMovix will display it during playback.
&lt;p&gt;Additional background movies can be installed. However, this is not as simple as a few mouse clicks. The background movies are stored in the emovix shared data folder (mostly &lt;i&gt;/usr/share/emovix&lt;/i&gt; or &lt;i&gt;/usr/local/share/emovix&lt;/i&gt;) under &lt;em&gt;backgrounds&lt;/em&gt;. So to add a background one has to copy the file to that folder.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Startup Behavior</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="m_labelKeyboardLayout">
        <property name="text">
         <string>Keyboard Layout:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>eMovix boot message language:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="m_comboBootMessageLanguage">
        <property name="toolTip">
         <string>Select the language of the eMovix help screens</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel5">
        <property name="text">
         <string>Default boot label:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="m_comboDefaultBootLabel">
        <property name="toolTip">
         <string>Select the default Linux kernel configuration</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;&lt;b&gt;eMovix Boot Labels&lt;/b&gt;
&lt;p&gt;eMovix provides a variety of different boot configurations which can be selected at boot time via a boot label (comparable to Lilo or Grub). The many different boot configurations mainly influence the Video output.
&lt;p&gt;The &lt;b&gt;default&lt;/b&gt;, &lt;b&gt;movix&lt;/b&gt;, or &lt;b&gt;MoviX&lt;/b&gt; labels start a general Vesa video driver.
&lt;p&gt;The &lt;b&gt;TV&lt;/b&gt; labels can be used to direct video to the TV output of the graphic board. eMovix provides TVout drivers for different brands of graphic boards.
&lt;p&gt;The &lt;b&gt;FB&lt;/b&gt; labels refer to configurations that start a Frame Buffer driver in different screen resolutions.
&lt;p&gt;The &lt;b&gt;AA&lt;/b&gt; labels make eMovix output the video through the ASCII-Art library which displays the picture in text mode through the usage of simple ASCII characters.
&lt;p&gt;The &lt;b&gt;hd&lt;/b&gt; label makes eMovix boot from the local harddisk instead of the medium. This can be used to prevent accidental starting of an eMovix medium.
&lt;p&gt;The &lt;b&gt;floppy&lt;/b&gt; label makes eMovix boot from the local floppy drive instead of the medium.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KComboBox" name="m_comboKeyboardLayout">
        <property name="toolTip">
         <string>Select the layout of the keyboard</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;The keyboard layout selected here will be used for eMovix commands such as controlling the media player.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Behavior After Playing</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="m_checkEject">
        <property name="toolTip">
         <string>Eject the disk after playing has finished</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked the disk will be ejected after MPlayer has finished.</string>
        </property>
        <property name="text">
         <string>E&amp;ject disk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkShutdown">
        <property name="toolTip">
         <string>Shutdown after playing has finished</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked the PC will be shut down after MPlayer has finished playing.</string>
        </property>
        <property name="text">
         <string>Sh&amp;utdown</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkReboot">
        <property name="toolTip">
         <string>Reboot after playing has finished</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked the PC will be rebooted after MPlayer has finished playing.</string>
        </property>
        <property name="text">
         <string>Re&amp;boot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">kcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_checkShutdown</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_checkReboot</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkReboot</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_checkShutdown</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
